src/containers/SearchContainer/index.js

```diff
@@ -0,0 +1,213 @@
+/* eslint-disable no-underscore-dangle */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { navigate } from '@reach/router';
+import { Box, Heading, Tabs, Tab, Text, TextInput } from 'grommet';
+import { Search as SearchIcon } from 'grommet-icons';
+
+import { Link, HighlightedText } from '../../components';
+import { useParams } from '../../hooks/use-params';
+
+const capitalize = text =>
+  text.substring(0, 1).toUpperCase() + text.substring(1);
+
+const categoryLabel = category =>
+  category === 'homepanels' ? 'Home' : capitalize(category);
+
+const Results = ({ searchTerm, results }) => {
+  if (results.length === 0) {

```

Comment:
I haven't tested it, but it is assuming results can't be undefined. should the message be displayed if `results` is undefined?

Reply:
results will always be defined (defaults or fails to [] as needed). It is marked required in the PropTypes for Result (line 65) as a safety.